v0.1/core/brainAtlasVersion.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/core/brainAtlas.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Definition of a brain atlas",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type",
+    "@id",
+    "alias",
+    "annotationSet",
+    "fullName",
+    "releaseDate",
+    "terminology",
+    "versionIdentifier",
+    "versionInnovation" 
+  ],
+  "properties": {
```

Comment:
Again the general comment about the type. To be consistent, we should always define the type for a property.

Reply:
cf my comment in the other PRs. Let's figure out what is best practice here and then change it across the schemas in a PR.
Sorry that I started this :see_no_evil: